src/main/java/com/vmtapp/enterprise/dto/Ticket.java

```diff
@@ -4,8 +4,8 @@
 
 public @Data
 class Ticket {
-    public String firstname;
-    public  String lastname;
+    public String firstName;
+    public  String lastName;
```

Comment:
Similar to the naming conventions of methods, classes should start with a lowercase and each following word should be capitalized.
Source: https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html